Library/Homebrew/system_config.rb

```diff
@@ -136,6 +149,7 @@ def dump_verbose_config(f = $stdout)
       f.puts "Perl: #{describe_perl}"
       f.puts "Python: #{describe_python}"
       f.puts "Ruby: #{describe_ruby}"
+      f.puts "System Ruby: #{describe_system_ruby}"
```

Comment:
The main difference is that Git uses the first one in the PATH and the Homebrew Ruby is differentiating from the Ruby above it by the fact it may not be the first in the PATH.